app/models/bag_man_request.rb

```diff
@@ -39,7 +39,7 @@ def cancel!(reason)
     unless cancelled
       transaction do
         update!(cancelled: true, cancel_reason: reason)
-        replication_transfer.cancel!(reason)
+        replication_transfer.cancel!(reason, last_error)
         send_cancel(replication_transfer)
       end
     end
```

Comment:
`last_error` is a field on the bag_man_request.  

The last_error could contain information about why the bag_man_request was cancelled; more context over always having the cancel_reason_detail be nil. The other readily-available option is to forego sharing that information with the from_node.

No, what the bag_man_request should do when cancelled isn't affected by whether the jobs are running or not.